.github/ISSUE_TEMPLATE/fma-request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: 📦 New Fleet-maintained app
+about: Request to add an app to the Fleet-maintained app catalog
+title: ''
+labels: ':product,#g-software,fma'
+assignees: ''
+
+---
+
+### Requestor
+
+- Application name: TODO
+- Application platform: TODO (macOS/Windows)
+
+---
+
+### Validation / QA
+
+- [ ] Outputs generated
+- [ ] Manifest name matches osquery `app.name`
+- [ ] Manifest version scheme matches osquery `app.short_bundle_version` version scheme
+- [ ] Manifest `unique_identifier` matches osquery `app.bundle_identifier`
```

Comment:
Probably makes sense to note these as macOS only given that the app platform was specified above?